src/index.test.ts

```diff
@@ -1,6 +1,14 @@
+import { Duplex } from 'stream';
 import { JsonRpcEngine } from 'json-rpc-engine';
+import PortStream from 'extension-port-stream';
+import { Runtime } from 'webextension-polyfill-ts';
```

Comment:
Nit: this could be changed to a type import to make it more clear we're just importing types,

e.g.

```suggestion
import type { Runtime } from 'webextension-polyfill-ts';
```

Reply:
I honestly had no idea this is a thing. :sweat_smile: